.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,14 +1,33 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Propose an idea for a feature or an enhancement
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
+Before submitting a feature request, please take some time to read and consider
+how your idea aligns with the WordPress philosophy.
 
```

Comment:
I just want to say that I find this a bit tricky. 
".....please take some time to read and consider
how your idea aligns with the WordPress philosophy."

As feature suggestions can also bring about new ideas that we have not thought about before.

Reply:
What do you suggest it says instead?

Reply:
I'd be happy leaving this out altogether. I don't want it block the improvements to the bug report template 🙂

Reply:
I am thinking let's skip that part for now.